libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1573,10 +1573,19 @@ void AP_OSD_Screen::draw_horizon(uint8_t x, uint8_t y)
     WITH_SEMAPHORE(ahrs.get_semaphore());
     float roll;
     float pitch;
+    bool inverted = false;
     AP::vehicle()->get_osd_roll_pitch_rad(roll,pitch);
     pitch *= -1;
-
-    //inverted roll AH
+    const float roll_deg = RAD_TO_DEG * roll;
+    // Are we inverted? then flash horizon line
+    if (abs(roll_deg) >= 90) {
+       inverted = true;
+    }
+    // Aviation style AH instead of Betaflight FPV style
+    if (check_option(AP_OSD::OPTION_AVIATION_AH)) {
```

Comment:
```suggestion
    if (inverted && check_option(AP_OSD::OPTION_AVIATION_AH)) {
```
?

Reply:
thanks...noticed that in the SITL testing this AM after pushing...fixed